pynamodb/__init__.py

```diff
@@ -7,4 +7,4 @@
 """
 __author__ = 'Jharrod LaFon'
 __license__ = 'MIT'
-__version__ = '3.2.1'
+__version__ = '3.2.2'
```

Comment:
as written this is a backwards-incompatible change for UTCDateTimeAttribute so this should not be included in a patch or minor release

Reply:
It should be backwards-compatible in that it writes a different type but will happily read either - just like the boolean change a while back.

Reply:
Oh, super fun, I hadn't seen that last time I was poking at the code. I can add migration code and docs as well, then.

Would you prefer I bump the version number in my PR, or just let you wait to merge this until next time you're going to do a major release?

Reply:
If it's ok with you I'd prefer to just wait and merge as we're about to release 3.3 or 4.0